zdm/cosmology.py

```diff
@@ -274,6 +274,39 @@ def dFnu_to_dEnu(z,alpha=0,bandwidth=1.e9):
 	dE = 4*np.pi*(dl(z))**2/(1.+z)**(2.-alpha)
 	return dE
 
+######### possible source evolution functions go here ##########
+
+def choose_source_evolution_function(which=0):
+	"""
```

Comment:
I tend to have a brief `Arguments` list in the doc string detailing
the parameters that may be input to the method.  These defines
their allowed type and a brief note on what they are.

In this example, it would be something like:

```
Arguments:
      which (bool, optional):  defines which function to use.  Default (0) is sfr_evolution
```

Reply:
But if you find writing such additional docs to be too
big of a pain, I am fine with that.  And I'll just do it 
from time to time as I touch the code.  :)

Reply:
I can also help with that :)

Reply:
I'll try to gradually add these in the code, and with future commits. Given that this will be an ongoing task for some time however, I'll ask that this gets accepted and I'll gradually fix things.